app/views/shared/_header.html.erb

```diff
@@ -0,0 +1,51 @@
+<nav class="bg-gray-800">
```

Comment:
- [ ]OPTIONAL: There is no direct requirement that says user should be able to log out, therefore this requirement is optional,  but I strongly suggest to add log out button for logged in users.